src/test/java/com/google/firebase/auth/FirebaseUserManagerTest.java

```diff
@@ -476,6 +476,16 @@ public void testListZeroTenants() throws Exception {
     checkRequestHeaders(interceptor);
   }
 
+  @Test
+  public void testDeleteTenant() throws Exception {
```

Comment:
Make sure the TENANT_NOT_FOUND is handled correctly (similar to getTenant)

Reply:
Done.